mate-panel/panel-toplevel.c

```diff
@@ -3127,6 +3126,18 @@ panel_toplevel_dispose (GObject *widget)
 
 	panel_toplevel_disconnect_timeouts (toplevel);
 
+	if (toplevel->priv->animation_start_time) {
+		g_date_time_unref (toplevel->priv->animation_start_time);
+		toplevel->priv->animation_start_time = NULL;
+	}
+
+	if (toplevel->priv->animation_end_time) {
+		g_date_time_unref (toplevel->priv->animation_end_time);
+		toplevel->priv->animation_end_time = NULL;
+	}
+
+	panel_toplevel_disconnect_timeouts (toplevel);
```

Comment:
Is this correct to use `panel_toplevel_disconnect_timeouts` 2 times?
It's already at line 3127.

Reply:
No it isn't. I'm going to remobe the 1st one.

Reply:
@raveit65 done.